test/integration_test/helpers.js

```diff
@@ -0,0 +1,25 @@
+(function() {
+  global.expect = require('chai').expect;
+  module.exports = {
+    sharedBucket: 'aws-sdk-js-integration',
+    AWS: require('../../lib/aws'),
+    createBuffer: function(size) {
```

Comment:
This method makes sense in cucumber where you are supposed to write 'human' parseable tests, but I think it's overkill for mocha. You could have constants for 1KB, 1MB if you want, then call `new Buffer` in the test. 

Then there's less code that can possibly go wrong :wink: